src/Log/Listener/CsvLogListener.php

```diff
@@ -13,16 +13,12 @@ public function log($sectionName, $targetUrl, RuleInterface $rule, $success) {
         $this->logs[] = array($sectionName, $targetUrl, $rule->getName(), $success ? 'Ok' : 'FAILED');
     }
 
-
     public function finish() {
         $handle = fopen($this->logFileName, 'w');
         foreach($this->logs as $log) {
-            fputcsv($handle, $log);
+            fwrite($handle, implode(',', $log) . PHP_EOL);
```

Comment:
Why did you change this?
fputcsv does a little more than turning an array into comma seperated lists. It also adds quotes where needed.